test/fixtures/everything_at_once/results.json

```diff
@@ -0,0 +1,24 @@
+{
+    "status": "error",
+    "message": "\nStacktrace:\n [1] error(::String) at ./error.jl:33\n [2] top-level scope at ./runtests.jl:14\n [3] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113\n [4] top-level scope at ./runtests.jl:14\n",
```

Comment:
@iHiD Are we okay with the full path (`/buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/Test/src/Test.jl:1113`) being used for the the Julia test file?

Reply:
The test runner specific paths are shortened. This path is part of the error message that you'd get everywhere (unless you build Julia from scratch).

Reply:
👍

Reply:
`/buildworker/worker/package_linux64/build/usr/share/` could be stripped without losing any valuable information, the rest is actually important, if you decide you'd rather not expose that path to the user.

Reply:
Yeah - strip that if you can pls :)

Reply:
Resolved by 
01b8de3